slang.yaml

```diff
@@ -0,0 +1,4 @@
+base_locale: de
+fallback_strategy: base_locale
```

Comment:
TBD. What language should be used if the device language is none of the supported language? German or English?